Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/AutoMineMod.java

```diff
@@ -33,9 +33,11 @@ public void onUpdate()
 			|| mc.objectMouseOver.getBlockPos() == null)
 			return;
 		if(Block.getIdFromBlock(mc.theWorld.getBlockState(
-			mc.objectMouseOver.getBlockPos()).getBlock()) != 0)
-			mc.gameSettings.keyBindAttack.pressed = true;
-		else
+			mc.objectMouseOver.getBlockPos()).getBlock()) != 0){
+			if(wurst.mods.autoToolMod.isActive())
+				AutoToolMod.setSlot(mc.objectMouseOver.getBlockPos());
+				mc.gameSettings.keyBindAttack.pressed = true;
```

Comment:
You just changed it in the online editor without looking at in Eclipse, didn't you?

1. You only removed one of the brackets.
2. There's an `else` behind it, so even if you had removed both brackets, this would still not compile.